core/Cargo.toml

```diff
@@ -23,7 +23,7 @@ std = ["math/std", "winter-utils/std"]
 math = { package = "winter-math", version = "0.6", default-features = false }
 crypto = { package = "miden-crypto", version = "0.2", default-features = false }
 winter-crypto = { package = "winter-crypto", version = "0.6", default-features = false }
-winter-utils = { package = "winter-utils", version = "0.6", default-features = false }
+winter-utils = { package = "winter-utils", version = "0.6.1", default-features = false }
```

Comment:
I had bump this, other wise `cargo test -p miden-vm` fails.

Reply:
I think you can just run `cargo update` and it will fix this.